generator/src/test/java/com/scottlogic/deg/generator/cucumber/AllOf.feature

```diff
@@ -37,7 +37,7 @@ Scenario: Running an 'allOf' request that contains a valid nested anyOf request
        | "1"  |
        | "11" |
 
-Scenario: Running an 'allOf' request that contains an invalid nested allOf request should fail with an error message
+Scenario: Running an 'allOf' request that contains an invalid nested allOf request should be successful
```

Comment:
If it's invalid why should it be accepted? Seems like some backwards wording in the test here

Reply:
Maybe instead of "be successful" we can say "produce an output containing only null"

Reply:
Hi, @fbromley and @cakehurstryan, can you pitch in here please? Thanks!

Also, @MattCline-SL, @fbromley is editting all these tests in branch updateContradictionTests as part of #349